src/FileIndexer.ts

```diff
@@ -148,11 +150,55 @@ export class FileIndexer {
     if (docstring.length > 0) {
       documentation.push(ts.displayPartsToString(docstring))
     }
+
     this.document.symbols.push(
-      new lsiftyped.SymbolInformation({ symbol: symbol.value, documentation })
+      new lsiftyped.SymbolInformation({
+        symbol: symbol.value,
+        documentation,
+        relationships: this.scipRelationships(declaration, symbol),
+      })
     )
   }
 
+  private scipRelationships(
```

Comment:
```suggestion
  private relationships(
```

nit: It seems a little weird to have scip in the method name; other methods don't seem to have this.

Reply:
Done. This was actually named `relationships` in my initial draft but I renamed to `scipRelationships` for consistency with `lsifSymbol` (which should be renamed into `scipSymbol` but I wanted to minimize unrelated diffs). Either way, I'm happy with `relationships`.